generated/nidaqmx/errors.py

```diff
@@ -45,18 +45,24 @@ def __init__(self, message, error_code, task_name=''):
             message (string): Specifies the error message.
             error_code (int): Specifies the NI-DAQmx error code.
         """
-        if task_name:
-            message = f'{message}\n\nTask Name: {task_name}'
-
-        super().__init__(message)
-
         self._error_code = int(error_code)
 
         try:
             self._error_type = DAQmxErrors(self._error_code)
         except ValueError:
             self._error_type = DAQmxErrors.UNKNOWN
 
+        # If message is empty, we try to put at least some information in it
+        if not message and self._error_type != DAQmxErrors.UNKNOWN:
+            message = f'Explanation could not be found for the requested status code.\n\nVerify that the requested status code is correct.\n\nError {self._error_type.name}'
```

Comment:
If the error is not in the enum, this will omit the text and log "\n\nTask Name: ..."

I think you should simplify this and ignore the enum. Users should very rarely encounter this situation, and if they do, the status code should be sufficient to debug the problem.

```suggestion
        # If message is empty, we try to put at least some information in it
        if not message:
            message = f'Explanation could not be found for the requested status code.\n\nVerify that the requested status code is correct.\n\nStatus Code: {self._error_code}'
```

Reply:
Changed.